lib/katello/tasks/delete_orphaned_content.rake

```diff
@@ -3,6 +3,8 @@ namespace :katello do
     User.current = User.anonymous_admin
     SmartProxy.with_content.reverse_each do |proxy|
       begin
+        ForemanTasks.async_task(Actions::Katello::CapsuleContent::RemoveUnneededRepos, proxy)
```

Comment:
I think this should be run within the remove orphans task prior to remove orphans and NOT for the master server.